sensing/imu_corrector/config/gyro_bias_estimator.param.yaml

```diff
@@ -1,6 +1,6 @@
 /**:
   ros__parameters:
     gyro_bias_threshold: 0.0015 # [rad/s]
-    velocity_threshold: 0.0 # [m/s]
-    timestamp_threshold: 0.1 # [s]
+    timer_callback_interval_sec: 0.5 # [sec]
+    straight_motion_ang_vel_upper_limit: 0.015 # [rad/s]
     data_num_threshold: 200 # [num]
```

Comment:
Did you test with this value? If using 0.5[sec] per data as a frequency, collecting 200 values sounds too large. Would like to know if there is a room for making this value smaller. 

(Of course it's a trade-off between robustness)

Reply:
Sorry the code is not clear.

The `data_num_threshold` (200) is used as just an upper limit. If even one data item is obtained, a bias is calculated and diagnostics are also output.

I was concerned that if this limit is not defined, the `gyro_bias_deque_` will get more and more.

By removing the `get_bias_std` function, this variable can also be removed, which would reduce confusion.

Reply:
I see, make sense (I also failed to read the code correctly).

But removing the threshold may not be a good idea, since the bias with only one data item may be unreliable.
IMO it's better to publish the diagnostics when enough data is gathered.

Reply:
Even the first time, there is some reliability as it corresponds to an average of 0.5 seconds. (The leftmost point on the graph in the pull request description field corresponds to the first point)

Also, it seems more legitimate to use stddev together with reliability rather than specifying a fixed number of times. (Although I deleted stddev...)

Reply:
> Also, it seems more legitimate to use stddev together with reliability rather than specifying a fixed number of times.

True :sweat_smile: 

Understood that the estimate from 0.5 seconds are accurate for most of the cases. Still I am a bit worried about making a false positives when picking unlucky unstable data for the initial data, but should be fine as the first data would likely to be a data when the vehicle is stopped. Thanks for the clarification!